### PR TITLE
add  isBestPrice condition

### PR DESCRIPTION
**Pull request by IncognitaDev**
#### What problem is this solving?

Add a new condition in product context that verify if the selling price of current seller is best price, is useful when using in marketplaces

#### How to test it?

Add `condition-layout.product` and pass  `{ "subject": "isBestPrice" }` in the conditions prop

[Workspace](https://luis--iaqua.myvtex.com/aerador-aquamix-trifasico-76/p)

#### Screenshots or example usage:

When is the best price: 
image

when is not best price: 

image

#### Related to / Depends on

I consider  this condition is more usefull if the [PR#38](https://github.com/vtex-apps/condition-layout/pull/38) is accept

### CODE REVIEW

File: react/ConditionLayoutProduct.tsx [hunk 1]
```diff
@@ -81,6 +83,16 @@ export const HANDLERS: Handlers<ContextValues, HandlerArguments> = {
...
   },
+  isBestPrice({ values }) {
+    const { sellers, priceRange } = values
+
+    const [sellerDefault] = sellers.filter((seller) => seller.sellerDefault)
+
+    const bestPrice = priceRange.sellingPrice.lowPrice
+    const currentPrice = sellerDefault?.commertialOffer.Price
+
+    return Boolean(currentPrice === bestPrice)
```
Comment (igorbrasileiro) [inline react/ConditionLayoutProduct.tsx:94]: This already returns a boolean, don't need the constructor.
```suggestion
    return currentPrice === bestPrice
```

**Review by igorbrasileiro — CHANGES_REQUESTED** [review]
Remember to add changelog

**Comment by IncognitaDev** [comment]
> Great job 🚀 ! Remember to add changelog.

Where I shoud put in changelog, after `[Unreleased]` tag ?

**Comment by igorbrasileiro** [comment]
> Where I shoud put in changelog, after `[Unreleased]` tag ?

Yes, sure.
```md
[Unreleased]
### Added
- <Message_Here>
```

**Pull request closed without merging**